about/list_of_features.rst

```diff
@@ -112,60 +130,57 @@ Vulkan renderer.
      those styles.
    - Dynamic fonts support oversampling to keep fonts sharp at higher resolutions.
    - Dynamic fonts support subpixel positioning to make fonts crisper at low sizes.
+   - Dynamic fonts support LCD subpixel optimizations to make fonts even crisper at low sizes.
    - Signed distance field fonts can be scaled at any resolution without
      requiring re-rasterization. Multi-channel usage makes SDF fonts scale down
      to lower sizes better compared to monochrome SDF fonts.
 
-- GPU-based particles with support for custom particle shaders.
+- GPU-based :ref:`particles <doc_particle_systems_2d>` with support for
+  :ref:`custom particle shaders <doc_particle_shader>`.
 - CPU-based particles.
 
 2D tools
 ^^^^^^^^
 
+- :ref:`TileMaps <doc_using_tilemaps>` for 2D tile-based level design.
 - 2D camera with built-in smoothing and drag margins.
 - Path2D node to represent a path in 2D space.
 
    - Can be drawn in the editor or generated procedurally.
    - PathFollow2D node to make nodes follow a Path2D.
 
-- 2D geometry helper class.
-- Line2D node to draw textured 2D lines.
+- :ref:`2D geometry helper class <class_Geometry2D>`.
 
 2D physics
 ^^^^^^^^^^
 
 **Physics bodies:**
 
 - Static bodies.
+- Animatable bodies (for objects moving only by script or animation, such as doors and platforms).
 - Rigid bodies.
-- Kinematic bodies.
+- Character bodies.
 - Joints.
 - Areas to detect bodies entering or leaving it.
 
 **Collision detection:**
 
-- Built-in shapes: line, box, circle, capsule.
+- Built-in shapes: line, box, circle, capsule, world boundary (infinite plane).
 - Collision polygons (can be drawn manually or generated from a sprite in the editor).
 
 3D graphics
 ^^^^^^^^^^^
 
-Vulkan renderer compatible with desktop and mobile platforms. Requires full support for
-Vulkan 1.0, with Vulkan 1.1 and 1.2 features optionally used.
-
 - HDR rendering with sRGB.
 - Perspective, orthographic and frustum-offset cameras.
-- When using the Vulkan Clustered backend (default on desktop), a depth prepass
-  is used to improve performance in complex scenes by reducing the cost of overdraw.
-
-- `OpenGL support planned for a future Godot 4.x release <https://godotengine.org/article/about-godot4-vulkan-gles3-and-gles2>`__.
-
-  - If you need OpenGL support, use Godot 3.x which remains supported.
+- When using the Forward+ backend, a depth prepass is used to improve
+  performance in complex scenes by reducing the cost of overdraw.
+- :ref:`doc_variable_rate_shading` on supported GPUs in Forward+ and Forward Mobile.
```

Comment:
The reference `doc_variable_rate_shading` is invalid, not sure what to link here.

Reply:
This is pending on https://github.com/godotengine/godot-docs/pull/6402 to be merged :slightly_smiling_face: